_posts/2017-12-30-new-years-rust-a-call-for-community-blogposts.md

```diff
@@ -0,0 +1,73 @@
+# New Year's Rust: A Call for Community Blogposts
+
+Tis the season for people and communities to reflect and set goals- and the Rust team is
+no different. Last week, we published [a blogpost about our accomplishments in 2017](rust-2017),
+and the teams have already begun brainstorming goals for next year. One thing that we know for
+certain is that Rust is commited to extending and improving our community organization and
+outreach- and this post is just the first of what we hope to be many iterations of new kinds of
+community feedback mechanisms.
+
+[rust-2017]: https://blog.rust-lang.org/2017/12/21/rust-in-2017.html
+
+Last year, the Rust team started a new tradition: defining a roadmap of goals for the 
+upcoming year. We leveraged our 
+[RFC process](https://github.com/aturon/rfcs/blob/roadmap-2017/text/0000-roadmap-2017.md)
+to solicit community feedback. While we got a lot of awesome feedback on that RFC, we'd 
+like to try something new **in addition to** the RFC process: a call for community blog posts
+for ideas of what the goals should be.
+
+## #Rust2018
+
+Starting today and running until the end of January we'd like to ask the community to write
+blogposts reflecting on Rust in 2017 and proposing goals and directions for Rust in 2018. 
+These can take many forms:
+
+- A post on your personal or company blog
+- A Medium post
+- A GitHub gist
+- Or any other online writing platform you prefer.
+
+We're looking for posts on many topics: 
+
+- Ideas for community programs
+- Language features
+- Documentation improvements
+- Ecosystem needs
+- Tooling enhancements
+- Or anything else Rust related you hope for in 2018 :D
+
+A great example of what we're looking for is this post by [@mgattozzi](https://github.com/mgattozzi):
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">New post is up for pushing to use <a href="https://twitter.com/rustlang?ref_src=twsrc%5Etfw">@rustlang</a> for WebAssembly:<br><br>Rust and the case for WebAssembly in 2018<a href="https://t.co/qEp91izSok">https://t.co/qEp91izSok</a></p>&mdash; Michael Gattozzi | マイケル・ガトジイ (@mgattozzi) <a href="https://twitter.com/mgattozzi/status/938115299290861569?ref_src=twsrc%5Etfw">December 5, 2017</a></blockquote>
+
+or this post by [Anonyfox](https://github.com/Anonyfox):
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">I just published “Rust in 2017” <a href="https://t.co/RYWUFsEzps">https://t.co/RYWUFsEzps</a></p>&mdash; Maximilian Stroh (@Hisako1337) <a href="https://twitter.com/Hisako1337/status/814435154764722176?ref_src=twsrc%5Etfw">December 29, 2016</a></blockquote>
```

Comment:
I know it's a nice visual contrast from the otherwise plaintext post, but I'm not sure it's a good idea to include these tweets. They don't use the suggested `#Rust2018` hashtag (obviously—they were written before that was a thing), and thus don't follow the instructions given in the very next paragraph. Maybe just link to the posts?

Reply:
yeah this is a good idea probably. i mostly just wanted to break it up visually but you're right :)